_gitbook/conventions/README.md

```diff
@@ -0,0 +1,3 @@
+# Conventions
+
+If you follow these conventions, your code will be more accessible to other developers. If you use a standard [coding style](coding_style.md), your project will be navigable and readable to others. If you write [documentation](documenting_code.md), others will understand the intent and behavior of your code.
```

Comment:
Happy to take a cue from one of the maintainers on this. 

I intentionally avoided "pushy" language because I wanted to avoid thin ice on style-related issues. But, if others agree that this tone is more fitting, I'm happy to change it!